src/getHumanAge.test.js

```diff
@@ -8,6 +8,38 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge()).toBeInstanceOf(Array);
+  });
+
+  it(`should return 0 animals age for 0 human age `, () => {
```

Comment:
You have 2 parameters, so better type it like "catAge" and "dogAge", for instance. It's my personal opinion.